### PR TITLE
Add authorization

### PR DESCRIPTION
**Pull request by polubis**

### CODE REVIEW

File: src/shared/ui/alert/Alert.tsx [hunk 1]
```diff
@@ -8,15 +8,15 @@ import { AlertProps } from '.';
...
 import csx from './Alert.scss';
 
-export const Alert = ({ idx, message, type, onClose }: AlertProps) => {
+export const Alert = ({ message, type = 'error', onClose }: AlertProps) => {
   const render = usePortal();
 
   const className = [csx.alert, csx[type]].join(' ');
 
   return render(
     <div className={className}>
```
Comment (marcinreszko) [inline src/shared/ui/alert/Alert.tsx:17]: Just a question: would it work with template string? I mean: className={`${csx.alert} ${csx[type]`}? If so, shouldn't we stick to one convention?

File: src/views/login/login-form/LoginForm.tsx [hunk 2]
```diff
@@ -4,13 +4,15 @@ import { NavLink } from 'react-router-dom';
...
+
 import { loginFormConfig } from '.';
 
 import csx from './LoginForm.scss';
 
 interface LoginFormProps {
   isDisabled: boolean;
-  onSubmit(login: string, password: string): void;
+  onSubmit(credentials: LogInPayload): void;
 }
```
Comment (marcinreszko) [inline src/views/login/login-form/LoginForm.tsx:17]: Shouldn't it be moved to models.ts?

File: src/shared/ui/alert/Alert.tsx (hunk 1 above)
Comment (polubis) [reply src/shared/ui/alert/Alert.tsx:17]: Y it will

File: src/views/login/login-form/LoginForm.tsx (hunk 2 above)
Comment (polubis) [reply src/views/login/login-form/LoginForm.tsx:17]: Y

**Pull request merged**